server/repository/src/migrations/v1_06_00/mod.rs

```diff
@@ -1,17 +1,20 @@
 use super::{version::Version, Migration};
 
 use crate::StorageConnection;
+
 mod contact_trace;
+mod master_list;
 
 pub(crate) struct V1_06_00;
 
 impl Migration for V1_06_00 {
     fn version(&self) -> Version {
-        Version::from_str("1.6.0")
+        Version::from_str("1.6.00")
```

Comment:
Is there a difference between "1.6.0" and "1.6.00" or does this results in the same `Version`? so this is just a style fix?

Reply:
Shouldn't be any difference... have changed back to .0 to match rest of the migrations